### PR TITLE
Add heartbeat webhook config

### PR DESCRIPTION
**Pull request by QuentinBisson**
## Checklist

I have:

- [x] Described why this change is being introduced
- [x] Separated out refactoring/reformatting in a dedicated PR
- [x] Updated changelog in `CHANGELOG.md`

### CODE REVIEW

File: service/controller/resource/alerting/heartbeatwebhookconfig/resource.go [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+		return nil, microerror.Mask(err)
+	}
+
+	labels := make(map[string]string)
+	for k, v := range key.AlertmanagerLabels() {
+		labels[k] = v
+	}
+
+	sendResolved := false
+	urlAddress, err := url.Parse(fmt.Sprintf("https://api.opsgenie.com/v2/heartbeats/%s/ping", key.HeartbeatName(cluster, config.Installation)))
```
Comment (TheoBrigitte) [inline service/controller/resource/alerting/heartbeatwebhookconfig/resource.go:95]: Please move this into a key package function

File: service/key/key.go [hunk 2]
```diff
@@ -278,3 +278,7 @@ func IsCAPICluster(obj metav1.Object) bool {
 func IngressClassName() string {
 	return "nginx"
 }
+
+func OpsgenieKey() string {
+	return "opsgenie.key"
```
Comment (TheoBrigitte) [inline service/key/key.go:283]: Can't we just have ?

```suggestion
	return "opsgenie"
```

**Review by TheoBrigitte — CHANGES_REQUESTED** [review]

File: service/controller/resource/alerting/heartbeatwebhookconfig/resource.go (hunk 1 above)
Comment (QuentinBisson) [reply service/controller/resource/alerting/heartbeatwebhookconfig/resource.go]: Solved

File: service/key/key.go (hunk 2 above)
Comment (QuentinBisson) [reply service/key/key.go:287]: We could, I named that because the value is mounted into a secret and we know what this field is about

Comment (TheoBrigitte) [reply service/key/key.go:283]: It feels redundant to call a key key to me.

**Pull request merged**